assets/js/script.js

```diff
@@ -52,10 +54,52 @@ function submitMovieHandler(event){
 
         var movie = movieInput.value;
         movieInput.value = "";
+        console.log(movie);
+
+        //getMovieTrailer(movie);
+        //AJZ calling OMDB api and passing it the movie title searched for
+        callOmdb(movie,omdbKey);
 
-        getMovieTrailer(movie);
     }
 }
 
+// AJZ working on function to call OMDB and return info on a movie
+var callOmdb = function(movie, apiKeyOmdb){
```

Comment:
you made the omdb key a global variable so you don't need to pass it into the function